src/content/roadmap/verkle-trees/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Verkle trees
+description: A high level description of Verkle trees and how they will be used to upgrade Ethereum
+lang: en
+template: staking
+emoji: ":money_with_wings:"
+image: ../../../assets/staking/leslie-pool.png
+alt: Leslie the rhino swimming in the pool.
+sidebarDepth: 2
+summaryPoints:
+  - Discover Verkle Trees
+  - Read why Verkle Trees are a useful upgrade for Ethereum
+---
+
+Verkle tries (a portmanteau of "Vector commitment" and "Merkle Tries") are a data structure that be used to upgrade Ethereum nodes so that they can stop storing large amounts of state data without losing the ability to validate blocks.
```

Comment:
Are we going with "tree" or "trie"?

To my understanding, both would be correct, with trie being a bit more of a specific type of data tree. Quick Google search seems to favor "Verkle tree"

Reply:
(For Particia Merkle Tries we've tended to use capitalized "Trie"... so not quite sure exactly what we want to settle on, just crave consistency 🙂)